tests/test_utils.py

```diff
@@ -48,6 +49,77 @@ def test_list_exchanges(capsys):
     assert re.search(r"^bittrex$", captured.out, re.MULTILINE)
 
 
+def test_list_timeframes(capsys):
+
+    args = [
+        "list-timeframes",
+    ]
+    pargs = get_args(args)
+    pargs['config'] = None
+    with pytest.raises(OperationalException,
+                       match=r"This command requires a configured exchange.*"):
+        start_list_timeframes(pargs)
+
+    # Test with --config config.json.example
+    args = [
+        '--config', 'config.json.example',
+        "list-timeframes",
+    ]
+    start_list_timeframes(get_args(args))
```

Comment:
You will need to mock some of the exchange.
right now it's calling `load_markets()` which will make this test depend on the exchange being available

I'm not sure if `patch_exchange()` will work in this case - maybe you'll need to only patch specific parts...

Reply:
That's why I said I need a template for testing this subcommand... 😄

Reply:
well but "this" template may not work for other subcommands ...